packages/lending/src/protocols/aave-v2/lending-protocol.ts

```diff
@@ -79,6 +84,17 @@ export class LendingProtocol extends Protocol {
     this.reserveMap = reserveMap;
   }
 
+  async getReserveTokens() {
+    const reserveTokens: ReserveTokens[] = await fetchReservesData(this.id, this.chainId);
+    return reserveTokens;
+  }
+
+  async getReserveTokensOnChain() {
```

Comment:
Is it better to rename

- getReserveTokens -> getReserveTokensFromCache
- getReserveTokensOnChain -> getReserveTokens


So that it would match the description on the ticket

Reply:
thanks for nameing suggestion, fixed